README.md

```diff
@@ -85,24 +85,25 @@ To change content view you should choose desired content controller and hide men
     menuContainerViewController.hideMenu()
  ```
 
-To customize animation for menu opening or closing you should override ```menuTransitionOptions()``` method that is available in ```MenuContainerViewColtroller``` class.
+To customize animation for menu opening or closing you should call ```setMenuTransition(options:)``` method that is available in ```MenuContainerViewColtroller``` class. Initial setup could be done, for example, on controller's ```viewDidLoad()```.
  ```swift
-override func menuTransitionOptions() -> TransitionOptions? {
-    var options = TransitionOptions(duration: 0.4, contentScale: 0.9)
-    options.useFinishingSpringOption = false
-    options.useCancelingSpringOption = false
-    return options
+override func viewDidLoad() {
+    super.viewDidLoad()
+    let screenSize: CGRect = UIScreen.main.bounds
+    let params = TransitionOptions(duration: 0.4, visibleContentWidth: screenSize.width / 6)
+    setMenuTransition(options: params)
```

Comment:
Let's rename this method to `setMenuTransitionOptions( _ options: TransitionOptions)`. It seems it looks more convenient. Usage: 
````swift
let options = TransitionOptions(duration: 0.4, visibleContentWidth: screenSize.width / 6)
setMenuTransitionOptions(options)
````